### PR TITLE
Update fluent forward receiver README

### PR DESCRIPTION
**Pull request by flands**
Also set default `endpoint` to ease configuration.

### CODE REVIEW

File: receiver/fluentforwardreceiver/receiver.go [hunk 1]
```diff
@@ -29,6 +29,8 @@ import (
 // FluentBit and increase throughput.
 const eventChannelLength = 100
 
+const defaultBindEndpoint = "0.0.0.0:8006"
```
Comment (tigrannajaryan) [inline receiver/fluentforwardreceiver/receiver.go:32]: Is this a known port number for Forward protocol?

**Comment by tigrannajaryan** [comment]
@flands The build failed.

**Comment by flands** [comment]
@tigrannajaryan yes on my list

**Comment by tigrannajaryan** [comment]
@flands please fix the conflicts and the build.

File: receiver/fluentforwardreceiver/receiver.go (hunk 1 above)
Comment (bogdandrutu) [reply receiver/fluentforwardreceiver/receiver.go:32]: Based on the documentation it does not seem to be the default, the default is `5170` https://docs.fluentbit.io/manual/pipeline/inputs/tcp

File: receiver/fluentforwardreceiver/receiver.go [hunk 2]
```diff
@@ -59,7 +61,11 @@ func (r *fluentReceiver) Start(ctx context.Context, _ component.Host) error {
 
 	r.collector.Start(receiverCtx)
 
-	listenAddr := r.conf.ListenAddress
+	if r.conf.ListenAddress != nil {
+		listenAddr := r.conf.ListenAddress
+	} else {
+		listenAddr := defaultBindAddress
+	}
```
Comment (bogdandrutu) [inline receiver/fluentforwardreceiver/receiver.go:68]: You should not have this custom logic, instead change the factory `createDefaultConfig` to set the value by default.

**Comment by bogdandrutu** [comment]
ping @flands

**Pull request closed without merging**